ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -257,36 +32,12 @@ const VariantSearchFormContent = React.memo((
       </Grid>
     </Header>
     <Header content="Customize Search:" />
-    <VariantSearchFormPanels esEnabled={esEnabled} panels={getPanels(hasHgmdPermission, inheritance, datasetTypes)} />
+    <VariantSearchFormPanels />
   </div>
 ))
 
 VariantSearchFormContent.propTypes = {
```

Comment:
all the code removed below is needed only for the `VariantSearchFormPanels` and has been moved to that file